src/v0/timetracking.ts

```diff
@@ -173,6 +197,40 @@ export class Timetracking extends ThBaseHandler {
     }
   }
 
+  async getConfiguration (): Promise<TimetrackingConfigurationResponse> {
+    const uri = this.uriHelper.generateBaseUri('/configurations')
+    try {
+      const response = await this.http.getClient().get(uri)
+      if (response.status !== 200) { throw new TimetrackingConfigurationFetchFailed(undefined, { status: response.status }) }
+
+      return {
+        data: response.data.results,
```

Comment:
Since the config is a single object we want to return it directly, and not be wrapped in an array.
```suggestion
        data: response.data.results[0],
```

Reply:
As when i did a RQ i got an array of three objects, i supposed that at the end its an array of objects. What do you think?

Reply:
True. Forget about it.